src/test/java/org/opensearch/knn/index/query/KNNQueryFactoryTests.java

```diff
@@ -85,6 +86,98 @@ public void testCreateLuceneDefaultQuery() {
         }
     }
 
+    public void testLuceneFloatVectorQuery() {
+        Query actualQuery1 = KNNQueryFactory.create(
```

Comment:
Can you add a function to create the query as it is duplicated multiple times

Reply:
It has a different request everytime. The create request itself is using a builder so wrapping it up with a function is moot